addons/website_event_track_live/views/event_track_templates_page.xml

```diff
@@ -47,7 +47,7 @@
             </a>
         </li>
     </xpath>
-    <xpath expr="//a[@href='#track_list']" position="attributes">
+    <xpath expr="//a[@aria-controls='track_list']" position="attributes">
```

Comment:
As the `href` now become a translatable attribute, we cannot rely on it to do xpath anymore. (because it may change depending on the lang)

Reply:
> As the `href` now become a translatable attribute, we cannot rely on it to do xpath anymore. (because it may change depending on the lang)

yes but this targets an ID.. it is not supposed to be tanslated

Reply:
I don't understand. The xpath was targeting `//a[@href='#track_list']` which matches `<a href="#track_list"/>` and we can't use xpath on a attribute that is translatable.